### PR TITLE
Chore: Don't import from outside of grafana-ui

### LINKED ISSUE #81148: `@grafana/ui` imports modules from Grafana
I noticed by chance a couple of modules in `@grafana/ui` that import modules from Grafana. In particular:

1. `packages/grafana-ui/src/components/Select/SelectBase.test.tsx` has:
```
import { selectOptionInTest } from '../../../../../public/test/helpers/selectOptionInTest';
```

2. `packages/grafana-ui/src/components/uPlot/utils.test.ts` has:
```
import { preparePlotFrame } from '../../../../../public/app/core/components/GraphNG/utils';
```

They are both test files, so maybe it is not an issue, but I would expect a `@grafana` package—`ui`, but also `data`, etc.—not to import files that live within Grafana itself. Feel free to ignore and close this issue if I'm wrong!

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- stops importing code from outside of grafana-ui

**Why do we need this feature?**

- it's fine because these are test files, but if this was production code it would break horribily 😅 

**Who is this feature for?**

- everyone

**Which issue(s) does this PR fix?**:

Fixes #81148 

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/uPlot/utils.test.ts
Comment (JoaoSilvaGrafana) [inline packages/grafana-ui/src/components/uPlot/utils.test.ts:4]: I'm assuming this was checked with data viz team, but this export says
```
// Graveyard: exported, but no longer used internally
// These will be removed in the future
```

**Review by JoaoSilvaGrafana — APPROVED** [review]
Nice 👏 
Just a question that data viz team must have more context on, approving from the `Select` side anyway

File: packages/grafana-ui/src/components/uPlot/utils.test.ts
Comment (ashharrison90) [reply packages/grafana-ui/src/components/uPlot/utils.test.ts:4]: naw i haven't checked, but i'll wait for a review from them 👍

**Review by nmarrs — APPROVED** [review]
After discussing this with the team this change _should_ be ok 😬

**Pull request merged**
